### PR TITLE
N4A optional exercises

### PR DESCRIPTION
**Pull request by chrisakker**
### Proposed changes

Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue using one of the [supported keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) here in this description (not in the title of the PR).

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginxinc/nginx-azure-workshops/blob/main/CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation ([`README.md`](https://github.com/nginxinc/nginx-azure-workshops/blob/main/README.md) and [`CHANGELOG.md`](https://github.com/nginxinc/nginx-azure-workshops/blob/main/CHANGELOG.md))

### CODE REVIEW

File: labs/lab3/readme.md [hunk 1]
```diff
@@ -803,7 +902,7 @@ So why use ports 9001 and 9002 for the NIC Dashboards?  Will this work on port 8
       access_log off;
       
       location = /dashboard.html {
-      #return 200 "You have reached /nic2dashboard.";
+      #return 200 "You have reached /nic1dashboard.";
```
Comment (sdutta9) [inline labs/lab3/readme.md:905]: In the commented return statement it should be nic2dashboard. Not a big deal.

File: labs/lab5/readme.md [hunk 2]
```diff
@@ -740,9 +742,9 @@ Cafe Nginx - Split 3 ways: DockerVM, AKS1, AKS2
 
 No worries, you comment out the `aks2_ingress` in the Split Config, and his 1% Live traffic is now going somewhere safe, as soon as you Submit your Nginx Configuration!
 
-But don't be surprised - in a few days he will ask again to send traffic to AKS2, and you can begin the Split migration process, this time from AKS1 to AKS2.  
+But don't be surprised - in a few days he will ask you again to send some live traffic to AKS2, and you can begin the Split migration process, this time from AKS1 >> AKS2.  
 
->>Now you've reached the Ultimate Kubernetes Application Solution, `Mutli Cluster Load Balancing, Active/Active, with Dynamic Split Ratios.` No one else can do this for your app team this easily, it's just Nginx!  
+>>Now you've reached the Ultimate Kubernetes Application Solution, `Nginx Mutli Cluster Load Balancing, Active/Active, with Dynamic Split Ratios.` No one else can do this for your app team this easily, it's just Nginx!
```
Comment (sdutta9) [inline labs/lab5/readme.md:747]: Typo. `Mutli` instead of `Multi`

File: labs/lab5/readme.md [hunk 3]
```diff
@@ -760,13 +762,15 @@ Using the HTTP Split Clients module from Nginx can provide multiple traffic mana
...
+<br/>
+
+## Optional: Configure NGINXaaS for Azure for Redis traffic
 
 NGINXaaS | Redis
 :-------:|:------:
 ![Redis](media/nginx-azure-icon.png)| ![Redis](media/redis-icon.png)
 
-In this exerices, you will use Nginx for Azure to expose the `Redis Leader Service` running in AKS Cluster #2. As Redis communicates with TCP instead of HTTP, the Nginx Stream Context will be used.  Following Nginx Best Practices, and standard Nginx folders/files layout, the `TCP Stream context` configuration files will be created in a new folder, called `/etc/nginx/stream/`.
+In this exerice, you will use Nginx for Azure to expose the `Redis Leader Service` running in AKS Cluster #2. As Redis communicates with TCP instead of HTTP, the Nginx Stream Context will be used.  Following Nginx Best Practices, and standard Nginx folders/files layout, the `TCP Stream context` configuration files will be created in a new folder, called `/etc/nginx/stream/`.
```
Comment (sdutta9) [inline labs/lab5/readme.md:773]: Typo `exerice` instead of `exercise`

File: labs/lab3/readme.md (hunk 1 above)
Comment (chrisakker) [reply labs/lab3/readme.md:905]: Fixed.

File: labs/lab5/readme.md (hunk 2 above)
Comment (chrisakker) [reply labs/lab5/readme.md:747]: Fixed.

File: labs/lab5/readme.md (hunk 3 above)
Comment (chrisakker) [reply labs/lab5/readme.md:773]: Fixed.

**Review by sdutta9 — APPROVED** [review]
Looks good.

**Pull request merged**